siphon/ncss.py

```diff
@@ -2,6 +2,8 @@
 from io import BytesIO
 
 import numpy as np
+from os import remove
+import atexit
```

Comment:
For cleanliness, can you move `atexit` import to the first line, and the `from os` to just under `from io`?